packages/process/src/node/terminal-process.ts

```diff
@@ -45,6 +45,12 @@ export enum NodePtyErrors {
     ENOENT = 'No such file or directory'
 }
 
+// Max size of a data batch. Note that this value can be exceeded by ~4k (chunk sizes seem to be 4k at the most)
+export const BATCH_MAX_SIZE = 200 * 1024;
+
+// Max duration to batch data before sending it to the renderer process.
+export const BATCH_DURATION_MS = 16;
```

Comment:
While testing this, I've noticed that 16 ms is way too short of a delay. Every other action was heavily delayed, often until the command running in the terminal has ended. Changing this to `64` has improved this behavior dramatically, basically enabling normal app interaction.